### DISCUSSION TITLE
In fabric.js v6.4.3 the `fabric.FabricObject` property does not have `group.removeAll()` `group.getObjects()`, is there any way I can replace it?

### DISCUSSION THREAD

**Discussion by hqw567** [fabricjs/fabric.js#10223] [Q&A]
In fabric.js v6.4.3 the `fabric.FabricObject` property does not have `group.removeAll()` `group.getObjects()`, is there any way I can replace it?

**Comment by asturur** [comment]
getObjects is part of the canvas/Group api, while removeAll is part of the group only. They are still there.
